lib/ipmi_sel.c

```diff
@@ -1955,8 +1955,13 @@ ipmi_sel_print_std_entry(struct ipmi_intf * intf, struct sel_event_record * evt)
 				(trigger_reading==(int)trigger_reading) ? 0 : 2,
 				trigger_reading);
 		if (threshold_reading_provided) {
+			bool ascending_threshold = (evt->sel_type.standard_type.event_data[0] & 0xf) % 2;
```

Comment:
For consistency with the IPMI Specification I would suggest to call this `going_high`.

I know that the original code, aside from the variable, is not yours, but please update it
to no use raw magic numbers, such as `0xf` and `0`. There is already a macro for `0xf`,
and the `0` and `2` can be addressed with a comment, like this:

```
/* According to Table 29-6, Event Data byte 1 contains,
   among other info, the offset from the Threshold type code.
   According to Table 42-2, all even offsets are 'going low', and
   all odd offsets are 'going high' */
bool going_high = (evt->sel_type.standard_type.event_data[0]
                   & EVENT_OFFSET_MASK) % 2;
```
Other than this, it's all fine. Thanks for the change.